openedx/core/lib/block_structure/transformers.py

```diff
@@ -1,9 +1,11 @@
 """
 Module for a collection of BlockStructureTransformers.
 """
+from functools import reduce as functools_reduce
```

Comment:
Oh cool, TIL. I think we'll try the for-loop implementation for readability, but good to know anyways - thanks!